pkgs/development/tools/misc/sysbench/default.nix

```diff
@@ -16,6 +16,8 @@ stdenv.mkDerivation rec {
     sha256 = "1sanvl2a52ff4shj62nw395zzgdgywplqvwip74ky8q7s6qjf5qy";
   };
 
+  configureFlags = [ "--with-gcc-arch=unknown" ];
```

Comment:
```suggestion
  configureFlags = [ "--without-gcc-arch" ];
```

Reply:
Actually, does it work if we set
```suggestion
  configureFlags = [ "--with-gcc-arch=${targetPlatform.config}" ];
```

Reply:
@jacereda Did you try the second option?

Reply:
I'll try in a moment, but that should be `stdenv.targetPlatform.config`

Reply:
That expands to `x86_64unknown-linux-gnu`. The configure script tries to pass that to gcc via `-march`/`-mcpu`/`-m` and in all those cases it won't work, so the end result is it won't pass any of those flags and behave the same as `--without-gcc-arch`. 

I guess something like `stdenv.targetPlatform.platform.gcc.arch` would be more appropriate?

Reply:
In any case, isn't `cc-wrapper` dealing with setting whatever flags it considers appropriate according to the configuration? In that case, I guess the right thing here is `--without-gcc-arch`.

Reply:
And now this leaves me wondering, there're quite a bunch of packages using `hostPlatform.avx*Support`/`hostPlatform.platform.gcc.arch` and more like that, where I think they should be inspecting the target platform (or nothing at all).

Reply:
They should definitely.